server/routes/transfers.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * @file Defines all routes for the transfers route.
+ */
+
+const express = require('express');
+const axios = require('axios');
+const {
+  retrieveTransfersByUserId,
+  createTransfer,
+  addTransferInfo,
+} = require('../db/queries');
+const { asyncWrapper } = require('../middleware');
+
+const router = express.Router();
+
+const { PLAID_CLIENT_ID, PLAID_SECRET_SANDBOX } = process.env;
+
+/**
+ * creates a transfer intent for Transfer UI and retrieves a transfer_intent_id
+ *
+ * @param {string} subscriptionAmount the amount of the transfer.
+ * @returns {Object}  transfer intent response
+ */
+
+router.post(
+  '/transfer_ui',
+  asyncWrapper(async (req, res) => {
+    try {
+      const { userId, subscriptionAmount } = req.body;
+      const transIntentCreateRequest = {
+        client_id: PLAID_CLIENT_ID,
+        secret: PLAID_SECRET_SANDBOX,
+        mode: 'PAYMENT',
+        amount: subscriptionAmount.toFixed(2),
+        ach_class: 'ppd',
+        description: 'foobar',
+        user: {
+          legal_name: 'Linda Woo',
+        },
+      };
+      let transferIntentId;
+
+      const transferIntentCreateResponse = await axios.post(
```

Comment:
similar to the above, is there any reason we can't use the plaid client for this file?

Reply:
none yet for transfers